js/skel.min.js

```diff
@@ -0,0 +1,2 @@
ocument,a=n,f=/^loaded|^c/.test(u[r]);return u[e](a,o=function(){u[i](a,o),t()}),function(e){f?e():s.push(e)}}):!function(e,t){_.DOMReady=t()}(t,function(t){function s(e){E=1;while(e=o.shift())e()}var o=[],u,a=!1,f=document,l=f.documentElement,c=l.doScroll,h=n,p=e,d="onreadystatechange",m=r,b=c?/^loaded|^c/:/^loaded|c/,E=b.test(f[m]);return f[p]&&f[p](h,u=function(){f[i](h,u,a),s()},a),c&&f.attachEvent(d,u=function(){/^c/.test(f[m])&&(f.detachEvent(d,u),s())}),t=c?function(e){self!=top?E?e():o.push(e):function(){try{l.doScroll("left")}catch(n){return setTimeout(function(){t(e)},50)}e()}()}:function(e){E?e():o.push(e)}}),document[s]?_[s]=function(e){return document[s](e)}:_[s]=function(e){var t=document;return t[o]?t[o](("."+e.replace(" "," .")).replace(/\.([0-9])/,".\\3$1 ")):[]},Array.prototype[u]?_[u]=function(e,t){return e[u](t)}:_[u]=function(e,t){if(typeof e=="string")return e[u](t);var n,r=t?t:0,i;if(!this)throw new TypeError;i=this.length;if(i===0||r>=i)return-1;r<0&&(r=i-Math.abs(r));for(n=r;n<i;n++)if(this[n]===e)return n;return-1},Object.keys?_.iterate=function(e,t){if(!e)return[];var n,r=Object.keys(e);for(n=0;r[n];n++)t(r[n])}:_.iterate=function(e,t){if(!e)return[];var n;for(n in e)Object.prototype.hasOwnProperty.call(e,n)&&t(n)}},initAPI:function(){var e="vars",t="match",n="deviceType",r="replace",i="_",s,o,u=navigator.userAgent;_[e].IEVersion=u[t](/MSIE ([0-9]+)\./)?RegExp.$1:99,_[e][n]="other",o={ios:"(iPad|iPhone|iPod)",android:"Android",mac:"Macintosh",wp:"Windows Phone",windows:"Windows NT"},_.iterate(o,function(r){u[t](new RegExp(o[r],"g"))&&(_[e][n]=r)});switch(_[e][n]){case"ios":u[t](/([0-9_]+) like Mac OS X/),s=parseFloat(RegExp.$1[r](i,".")[r](i,""));break;case"android":u[t](/Android ([0-9\.]+)/),s=parseFloat(RegExp.$1);break;case"mac":u[t](/Mac OS X ([0-9_]+)/),s=parseFloat(RegExp.$1[r](i,".")[r](i,""));break;case"wp":u[t](/IEMobile\/([0-9\.]+)/),s=parseFloat(RegExp.$1);break;case"windows":u[t](/Windows NT ([0-9\.]+)/),s=parseFloat(RegExp.$1);break;default:s=99}_[e].deviceVersion=s,_[e].isTouch=_[e][n]=="wp"?navigator.msMaxTouchPoints>0:"ontouchstart"in window,s=document.cookie.split(";"),_.iterate(s,function(e){var t=s[e].split("=");if(t[0][r](/^\s+|\s+$/g,"")==_.lsc){_.lockState=t[1];return}})},init:function(e,t){var n="registerLocation",r="html",i="getElementsByTagName",s="head",o="body",u="plugins";_.initUtilityMethods(),_.initAPI(),e&&(window._skel_config=e);if(t){var a;_.iterate(t,function(e){window["_skel_"+e+"_config"]=t[e]})}_.initConfig(),_[n](r,document[i](r)[0]),_[n](s,document[i](s)[0]),_.DOMReady(function(){_[n](o,document[i](o)[0])}),_.initEvents(),_.poll();var a;_.iterate(_[u],function(e){_.initPluginConfig(e,_[u][e]),_[u][e].init()}),_.isInit=!0},preInit:function(){var e="getElementsByTagName",t="script",n="isConfigured",r=document[e](t);_.me=r[r.length-1],window._skel_config?_[n]=!0:(s=document[e](t),s=s[s.length-1].innerHTML.replace(/^\s+|\s+$/g,""),s&&(_[n]=!0)),_[n]&&_.init()}};return _.preInit(),_}();
```

Comment:
This character may get silently deleted by one or more browsers.<br>Line is too long.<br>Mixed double and single quotes.<br>Identifier 'config_breakpoint' is not in camel case.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '{' and instead saw 't'.<br>'n' is already defined.<br>Expected '{' and instead saw 'return'.<br>Confusing use of '!'.<br>Expected '{' and instead saw 'r'.<br>Expected '{' and instead saw 'i'.<br>Use '===' to compare with '0'.<br>Too many errors. (100% scanned).